README.md

```diff
@@ -24,7 +24,7 @@ This scripts automatically sets up the project automatically and starts the appl
 ### STARTING THE APP
 Subsequently, if you need to start the application after the initial setup has completed you can run the command:
 ```
-./start.sh
+bin/start.sh
 ```
```

Comment:
All shell scripts should be in a single directory called scripts `Acceptance criteria`. 
Did you go through this on `Pivotal Tracker?` @ayodelevm

Reply:
Also, you didn't tag anyone on this PR, I will assume it's not ready... 
Do test everything out though

Reply:
@andela-sjames didn't take cognisance of that, used bin because I felt it's more popular to use bin... will fix that... Thanks...

Reply:
@parseendavid I think it will be better to raise this concern on the slack channel, so that people can get involved

Reply:
@andela-sjames PR fixed